### ISSUE TITLE
Cerrar el sistema actuel del CVU hasta que se arreglen los problemas

### ISSUE THREAD

**Issue by Morisset** [wlmb/CVU#29]
Creo muy importante que se cierre el sistema del CVU y que en su lugar se deja un aviso para explicar que tal como esta hecho no se puede usar y que hay gente trabajando en mejorarlo.
Seria para evitar que mas colegas se vean confrontados a los problemas que conocemos (mala migración, listas incompletas o erróneas, etc) y que tengan que volver a capturar la información una ves que el sistema estará en un estado aceptable.

**Comment by wlmb** [comment]
Estoy de acuerdo con @Morisset . Es importante cerrar el sistema, o al menos poner una indicación **muy visible** de que la versión actual es experimental, sujeta a cambios, y que su uso es para ejercitar al sistema durante un periodo de prueba, durante el cual se corre el riesgo de que la información se pierda.

Una vez hecha esta advertencia, sugiero una nueva ruta de liberación del sistema:

 1. Seguir usando esta plataforma para que usuarios y desarrolladores se mantengan en estrecha comunicación.

 1. Que los desarrolladores empleen el sistema de captura del CVU como si fueran usuarios, llenando información correspondiente a algunos cvu's viejos grandes, quizás de algunos investigadores eméritos, para que se enfrenten a las dificultades de su interface (para que vean lo que se siente :)  y la mejoren hasta que sea un sistema que ellos sientan que podrían usar gustozos si tuvieran que llenar sus propios cvu's. 

  1. Luego, reclutaría un pequeño grupo de voluntarios, preferentemente críticos, quisquillosos, obsesivos, quizás ofreciéndoles algún estímulo a cambio de su ingrata labor, para seguir probándolo y buscando sus fallas, haciendo sugerencias para su mejora. 

  1. Cuando dichos voluntarios se hallen convencido de la usabilidad del sistema, entonces invitaría a la comunidad a probarlo.

  1. Mientras tanto, completaría la migración de los datos desde el viejo cvu al nuevo cvu. Para ello, habría que sacrificar algunos aspectos que conacyt desea en el nuevo cvu, como el que todos los registros estén completos, con información normalizada de acuerdo a ciertos catálogos. 

  1. Completar los registros y normalizar la información es una tarea que deberá realizarse pero gradualmente y no como punto inicial del nuevo cvu. Es mejor información incompleta que borrón y cuenta nueva. 

  1. Una vez que se hayan concluido los pasos previos, mostrando que el sistema es usable, entonces lo liberaría.

  1. Al liberarlo, conacyt deberá tomar el compromiso de que la información capturada será preservada, no se perderá y por tanto no debera ser recapturada.

  1. Aún liberado, el sistema será imperfecto y habrá quien encuentre fallas o tenga sugerencias de mejoras. 

  1. Conacyt deberá seguir empleando esta plataforma o alguna plataforma similar para recabar reportes de fallas y sugerencias y darles seguimiento.

  1. Las mejoras subsecuentes deberán estar sujetas a la constricción de no perder información.
